std/fs/walk_test.ts

```diff
@@ -252,12 +252,11 @@ testWalk(
   async function symlink(): Promise<void> {
     assertReady(6);
     const files = await walkArray("a");
-    assertEquals(files.length, 2);
+    assertEquals(files.length, 3);
```

Comment:
shouldn't this one remain unchanged?

Reply:
I think `includeDirs` was off by default back when this test was written, so now it includes `"a"` itself as well.